test/typescript/styling-comparison.spec.tsx

```diff
@@ -46,3 +46,33 @@ const DecoratedNoProps = decorate<{}>(
 const sfcElem = <DecoratedSFC text="Hello, World!" variant="title" color="secondary" />;
 const classElem = <DecoratedClass text="Hello, World!" variant="title" color="secondary" />;
 const noPropsElem = <DecoratedNoProps />;
+
+// This is the "scenario 2" example straight from the doc, then invoked:
```

Comment:
There is no scenario 2 now, maybe just provide a link or else delete the comment.